src/build/builtin_compiler/main.zig

```diff
@@ -268,7 +268,6 @@ fn compileModule(
     module_env.* = try ModuleEnv.init(gpa, source);
     errdefer module_env.deinit();
 
-    module_env.common.source = source;
```

Comment:
Was this removal intentional?

Reply:
Kind of. I removed it because the `ModuleEnv.init` already does it and I wanted to test if removing it breaks anything. But I forgot that I actually committed it.
@bhansconnect Could you merge this as is?